examples/finetuning/dpo_tic_tac_toe/pyproject.toml

```
@@ -0,0 +1,26 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools >= 64", "setuptools-scm>=8"]
+
+[tool.setuptools_scm]
+# NAT uses the --first-parent flag to avoid tags from previous releases which have been merged into the develop branch
+# from causing an unexpected version change. This can be safely removed if developing outside of the NAT repository.
+git_describe_command = "git describe --long --first-parent"
+root = "../../.."
+
+[project]
+name = "nat_dpo_tic_tac_toe"
+dynamic = ["version"]
+dependencies = [
+  "nvidia-nat[langchain]~=1.4",
+  "numpy>=1.24.0"
```

This is a different version of numpy than the one specified in the top-level pyproject.toml file, update to match.
```suggestion
  # langchain-aws pins numpy to 1.* for Python <3.12 -- remove once package is updated
  "numpy~=1.26; python_version < '3.12'",
  "numpy~=2.3; python_version >= '3.12'",
```